Paws4Me/HomeCollectionViewCell.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  HomeCollectionViewCell.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/04/12.
+//
+
+import UIKit
+
+class HomeCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var cellView: UIView!
```

Comment:
small change. Maybe make these private :)

Reply:
I agree, thank you 👍🏻